### PR TITLE
Upgrade to Decidim v0.21

### PR DESCRIPTION
**Pull request by laurajaime**
Changelog: https://github.com/decidim/decidim/blob/release/0.21-stable/CHANGELOG.md

This PR upgrades the application to Decidim latest release/0.21-stable.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,18 +2,17 @@ source "https://rubygems.org"
 
 ruby RUBY_VERSION
 
-DECIDIM_VERSION = { git: 'https://github.com/decidim/decidim.git', branch: '0.19-stable' }
```
Comment (tramuntanal) [inline Gemfile]: This PR should branch off from the v0.20 branch in https://github.com/CodiTramuntana/decidim-salt-app/pull/35, which, by  the way, is already been merged into `master`.
Can you merge `master` then into this branch please?

File: Gemfile [hunk 2]
```diff
@@ -2,18 +2,17 @@ source "https://rubygems.org"
...
-gem 'decidim-file_authorization_handler', git: "https://github.com/CodiTramuntana/decidim-file_authorization_handler.git", branch: "fix/organization_error_on_manage_new_participant"
+gem 'decidim-file_authorization_handler', git: "https://github.com/CodiTramuntana/decidim-file_authorization_handler.git"
 # gem 'decidim-consultations', "~> #{DECIDIM_VERSION}"
 
 gem 'delayed_job_active_record'
 gem 'daemons'
 gem "whenever", require: false
 
-gem 'puma', '~> 3.0'
+gem 'puma', '>= 4.3'
```
Comment (tramuntanal) [inline Gemfile]: Puma >= 5 does not daemonize anymore. Let's keep it under 5 for the moment
 ```suggestion
gem 'puma', '< 5'
```

**Review by tramuntanal — CHANGES_REQUESTED** [review]
I've left just one comment to be checked @LauraJaimeC

File: Gemfile (hunk 1 above)
Comment (laurajaime) [reply Gemfile]: Done!!

File: Gemfile (hunk 2 above)
Comment (laurajaime) [reply Gemfile]: Done!!

**Review by tramuntanal — APPROVED** [review]
Good Job!

**Pull request merged**